src/main/java/com/passbook/sparkeighteen/controller/profileController.java

```diff
@@ -0,0 +1,30 @@
+package com.passbook.sparkeighteen.controller;
+
+import com.passbook.sparkeighteen.peristence.POJO.ProfileRequest;
+import com.passbook.sparkeighteen.peristence.POJO.ProfileResponse;
+import com.passbook.sparkeighteen.service.ProfileService;
+import io.swagger.annotations.ApiOperation;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+
+@RequestMapping("/api/v1/user")
+@RestController
+public class profileController {
```

Comment:
Why do you need a separate controller for profile? You already have user controller, this code must go there